### DISCUSSION TITLE
Default concurrent requests limit

### DISCUSSION THREAD

**Discussion by jwodder** [tokio-rs/axum#2600] [Q&A] [answered]
### Summary

What, if any, is the maximum number of concurrent requests `axum` can serve by default?  A server I've written seems to only be capable of handling a maximum of about 20 request at once, and I'm trying to figure out why; more details in [this URLO post](https://users.rust-lang.org/t/axum-request-limits/106867).

### axum version

0.7.4

**Comment by davidpdrsn** [comment, marked as answer]
axum doesn’t enforce any such limit by default.

**Reply by jwodder** [reply]
I suspected as much.  Would it be too much trouble for you or a colleague to weigh in on my URLO post?
